File: Python/tests/test_igt_vpp.py
import pytest

from hbayesdm.models import igt_vpp


def test_igt_vpp():
    _ = igt_vpp(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_vpp.R
context("Test igt_vpp")
library(hBayesDM)

test_that("Test igt_vpp", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_vpp(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
